app/frontend/src/router/index.js

```diff
@@ -40,6 +40,17 @@ const router = new VueRouter({
         requiresAuth: true
       },
     },
+
+    // tempoarary place for admin page
+    {
+      path: '/admin',
+      name: 'Admin',
+      component: () => import(/* webpackChunkName: "admin" */ '@/views/Admin.vue'),
+      meta: {
+        requiresAuth: true
+      },
+    },
+
```

Comment:
You will get merge conflicts with #150 . You'll have to drop this chunk from your git branch.